src/Pos/Block/Network/Listeners.hs

```diff
@@ -37,8 +38,8 @@ import           Pos.WorkMode                (WorkMode)
 
 blockListeners
     :: (WorkMode ssc m)
-    => ([ListenerSpec m], OutSpecs)
-blockListeners = mergeLs
+    => m ([ListenerSpec m], OutSpecs)
```

Comment:
It is, but I have to fetch limit from DB somehow and after that put it to `convListener`